com.civicrm.hrjobroles/js/test/controllers/job-roles.controller.spec.js

```diff
@@ -33,6 +34,30 @@ define([
       dateValidation.dateFormats.push('DD/MM/YYYY');
     }));
 
+    describe('when user opens option editor', function () {
+      var onPopupFormSuccess;
+      var url = '/civicrm/admin/options/hrjc_level_type?reset=1';
+      var optionType = 'hrjc_level_type';
+
+      beforeEach(function () {
+        spyOn(crmAngService, 'loadForm').and.callFake(function () {
+          return {
+            on: function (event, callback) {
+              if (event === 'crmUnload') {
+                onPopupFormSuccess = callback;
+              }
+            }
+          };
+        });
+        initController();
+        ctrl.openOptionsEditor(url, optionType);
+      });
+
+      it('calls the crmAngService with the requested url', function () {
+        expect(crmAngService.loadForm).toHaveBeenCalledWith(url);
+      });
+    });
+
```

Comment:
The tests that check for `vm.DepartmentsData` and similar storages are missing. Please add them.